src/domain/app/sentry/utils.ts

```diff
@@ -1,8 +1,64 @@
 import * as Sentry from '@sentry/react';
+import { ErrorEvent, TransactionEvent } from '@sentry/types';
 import * as H from 'history';
+import isObject from 'lodash/isObject';
+import snakeCase from 'lodash/snakeCase';
 
 import { UserFieldsFragment } from '../../../generated/graphql';
 
+const SENTRY_DENYLIST = [
```

Comment:
Maybe the common items from the Python `DEFAULT_DENYLIST` could be included in the front-end's list too ("apikey", "password", etc): https://github.com/getsentry/sentry-python/blob/8094c9e4462c7af4d73bfe3b6382791f9949e7f0/sentry_sdk/scrubber.py#L14

Reply:
That's a very good suggestion. The DEFAULT_DENYLIST items are now included in the SENTRY_DENYLIST